src/botPage/view/tour/index.js

```diff
@@ -35,8 +36,15 @@ class Tour extends PureComponent {
                 setDoneCheck();
             }
         };
+        const shouldShowTourPopup = () => {
+            const dayHasPassed = () => {
```

Comment:
This function could be a one liner and more readable. It can be implemented in the following way 

`
const dayHasPassed = () => new Date(getStorage('closedTourPopup') || 0).setHours(24) < Date.now();
`

Reply:
For the `setHours()` function we should get the initial hours of the `Date` object, and then add 24 for it to add the hours. It would only add 1 hour if we use `setHours(24)` according to the [documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setHours). In that case it would still not be a one-liner, any ideas?